.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -28,12 +28,15 @@ For refactoring and code cleanup changes, exercise the code before and after the
 
 ### Proposed changelog entries
 
-- Entry 1: Issue, human-readable text
-- […]
+- \<replace with Issue ID if it exists or remove\>, human-readable text
```

Comment:
That pattern got me confused to be honest. I think giving an example would work better.

Reply:
Noting previously negative feedback I got in https://github.com/jenkinsci/jenkins/pull/7236#discussion_r996221993